packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -159,6 +174,13 @@ export type AutocompleteProps<
    * You will need to implement this function if your `option` items are objects.
    */
   getIsOptionEqualToValue?: (option: OptionType, value: OptionType) => boolean;
+
+  /**
+   * If this component is required to display a virtualized list of options, this value needs
+   * to be toggled to true.
+   * It is recommended if you're options are on the order of 10's of hundreds or more
```

Comment:
Can `isVirtualized` be changed after loading the component or is it a one-and-done kinda thing like with our table component?

In those cases, I think they should probably be separate components because the assumption in React is that props can be manipulated at runtime.

Although, I think we don't do that on our Table right now, but Table is also in Labs still.

Reply:
This automagically happened by formatter, most likely. I fixed it manually so it reads better.